.gitignore

```diff
@@ -17,7 +17,6 @@ src/aws-lambda/personalize-delete-resources/models
 src/aws-lambda/personalize-pre-create-campaigns/models
 src/aws-lambda/personalize-pre-create-campaigns/data
 src/aws-lambda/pinpoint-auto-workshop/pinpoint-templates
-local/*
```

Comment:
Why remove this? I suppose it was for a reason (I don't remember where)

Reply:
Hmmm, I don't how that happened, I'll add it back in.